meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -135,7 +156,22 @@ export class MongoEntityConverter {
 	}
 
 	public convertSegments(mongoSegments: MongoSegment[]): Segment[] {
-		return mongoSegments.filter((segment) => !segment.isHidden).map(this.convertSegment)
+		return mongoSegments.filter((segment) => !segment.isHidden).map(this.convertSegment.bind(this))
+	}
+
+	public convertToMongoSegment(segment: Segment): MongoSegment {
+		return {
+			externalId: '', // Todo: figure out where the value for this attribute is
+			isHidden: false, // Todo: figure out where the value for this attribute is
+			_id: segment.id,
+			name: segment.name,
+			rundownId: segment.rundownId,
+			_rank: segment.rank,
+		}
+	}
+
+	public convertToMongoSegments(segments: Segment[]): MongoSegment[] {
+		return segments.map(this.convertToMongoSegment.bind(this))
```

Comment:
Binding it to `this` was something @KvelaGorrrrnio suggested, and he has also done so on my Cliptool refactor PR, so i just went ahead and applied it here too. Doesn't seem to harm anything, and might fix some unforseen edge cases 🤷 

What effect does `this: void` have?
Have never seen that before.

Reply:
> might fix some unforseen edge cases

YAGNI

You can write line after line of code for thinks that might happen but never actually does.

The reason why it doesn't seem to harm anything is because the method you are binding isn't using `this`.
I agree with Krzysztof, there really isn't a need to bind it here.
@KvelaGorrrrnio what was your reasoning for this?

Reply:
The reason I suggested binding the this-context, was for use cases for when the converter is fully implemented.
Unless the `convertToMongoSegment` is a one-to-one mapping, private methods seems likely to be used. I know that YAGNI is often a good practice, but calling a function with the wrongly bound context is not caught by the transpiler.

If there is a majority vote against binding the context, then am I fine with removing it.